src/components/elements/inputs/boolean-list.js

```diff
@@ -39,7 +40,7 @@ export default ({ className, onCommit, property, value }) => {
               checked={isChecked(option.value)}
               disabled={property.disabled}
               onChange={e => onCommit(getValue(e.target.checked, option.value))}
-              name={property.id}
+              name={property.name}
```

Comment:
We're not using ```id``` attribute here, will that be an issue? Assume we can't because there will be multiple checkboxes/radio buttons?